### DISCUSSION TITLE
Feature Request: export settings and favorites

### DISCUSSION THREAD

**Discussion by jaxaCe** [jocmp/capyreader#653] [Feature Requests]
Currently you can only export the current feeds.
After some time, a few favorites collect and you might want to export them, e.g. to import them on a new device.
Therefore, it would be great if there was the possibility to export the favorites and maybe even everything at once (favorites, feeds, settings).

**Comment by jocmp** [comment]
hey @jaxaCe thanks for the suggestion. I have a very old, open issue for this exact idea here https://github.com/jocmp/capyreader/issues/331. I haven't done anything with it yet because I need to explore a good format - hopefully one that's used by other platforms - to import/export saves.

**Reply by jaxaCe** [reply]
Oh, I missed that. I just looked in the discussions.

Handy-News-Reader, for example, uses its own XML format
https://github.com/yanus171/Handy-News-Reader/issues/355
https://github.com/yanus171/Handy-News-Reader/issues/48

https://github.com/yanus171/Handy-News-Reader/commit/e0d1c2a720178e0b0bf86866bcc9ad27e0c852e6
https://github.com/yanus171/Handy-News-Reader/commit/487c55f9e9e33361f5ab443c93124b6cb58cc4c2
